test/python/circuit/library/test_evolved_op_ansatz.py

```diff
@@ -84,7 +84,7 @@ def test_insert_barriers(self):
     def test_empty_build_fails(self):
         """Test setting no operators to evolve raises the appropriate error."""
         evo = EvolvedOperatorAnsatz()
-        with self.assertRaises(ValueError):
+        with self.assertRaises(TypeError):
```

Comment:
See comment above -- this should probably remain the `ValueError` it was before 🙂

Reply:
It was throwing TypeError instead of ValueError based on the changes made, so will need to track that down.

Reply:
I think that's because of the removed `_check_configuration`, which previously threw the `ValueError` that `self.operators` is `None`. But with your current version, it tries to iterate over the `None` operators which throws a `TypeError`